lib/vrl/web-playground/netlify.toml

```diff
@@ -2,18 +2,16 @@
 # all contexts unless otherwise overridden by more specific contexts.
 [build]
   # Directory to change to before starting a build.
-  # This is where we will look for package.json/.nvmrc/etc.
+  # This is where we will build the site in web-playground/public.
   # If not set, defaults to the root directory.
   base = "lib/vrl/web-playground/"
 
   # Directory that contains the deploy-ready HTML files and
-  # assets generated by the build. This is relative to the base
-  # directory if one has been set, or the root directory if
-  # a base has not been set. This sample publishes the directory
-  # located at the absolute path "root/project/build-output"
-
+  # assets generated by the build.
   publish = "public/"
 
   # Default build command.
-  command = "cargo install --version 0.10.3 wasm-pack && wasm-pack build --target web --out-dir public"
+  command = "cargo install --force --version 0.10.3 wasm-pack && wasm-pack build --target web --out-dir public"
 
+  # Ignore everything except the base directory and changes to lib/vrl/stdlib
+  ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF . ../stdlib"
```

Comment:
I'd at least expand this to cover all of `lib/vrl` though `stdlib` has references to even more crates like `vector-common` so changes to these crates should also trigger a rebuild.

Personally I don't really think we need to skip builds.